README.md

```diff
@@ -177,6 +177,18 @@ const updated = await core.update()
 console.log('core was updated?', updated, 'length is', core.length)
 ```
 
+#### `const stats = await core.stat()`
+
+Get information about this core, such as its total size in bytes.
+
+The object will look like this:
+
+```js
+Stats {
+  size: 742
```

Comment:
what about just saying byteLength here? I think we should add all cheap metadata here also, like the length, fork etc (even tho we have it as props also)

Based on feedback from @andrewosh I think we should name the function info also (or similar if you have ideas)

Reply:
A really cool stat that would help users would be bytesUsed or something also, that infers how much data is actually used if using files (ie fileStat.blocks * 512 vs fileStat.size)

Reply:
That should really come from the `random-access-*` modules, no?